### PR TITLE
Config

### PR DESCRIPTION
**Pull request by KunSu**

### CODE REVIEW

File: 01-connecting-to-simulator.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# This software contains code licensed as described in LICENSE.
+#
+
+import simulation.config as config
+
+# Connects to the simulator instance at the ip defined by SIMULATOR_HOST, default is localhost or 127.0.0.1
+cf = config.Config()
+sim = cf.Simulator()
+
+# sim = lgsvl.Simulator(os.environ.get("SIMULATOR_HOST", "127.0.0.1"), 8181)
```
Comment (zijianguan0204) [inline 01-connecting-to-simulator.py]: why we comment out this line?

File: 03-raycast.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+# Included layers can be hit by the rays. Otherwise the ray will go through the layer
+layer_mask = cf.Raycast_Layer()
+
+# raycast returns None if the ray doesn't collide with anything
+# hit also has the point property which is the Unity position vector of where the ray collided with something
+hit = sim.raycast(p, right, layer_mask)
+if hit:
+  print("Distance right:", hit.distance)
+
+hit = sim.raycast(p, -right, layer_mask)
```
Comment (zijianguan0204) [inline 03-raycast.py:46]: is it supposed to be left instead of "-right"? or both are the same thing

File: 03-raycast.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+if hit:
+  print("Distance left:", hit.distance)
```
Comment (zijianguan0204) [inline 03-raycast.py:48]: since layermask can show what object it hits based on the return bits, shall we show that msg on the printed output?

File: 01-connecting-to-simulator.py (hunk 1 above)
Comment (KunSu) [reply 01-connecting-to-simulator.py]: Deleted. 
sim = cf.Simulator() already do the same job for this line of code.

File: 03-raycast.py (hunk 2 above)
Comment (KunSu) [reply 03-raycast.py:46]: So for raycasting in a 3D world, there are only 3 vectors x, y, and z which are the forward, right, and up. If we want to go left, then we are simply use -right. The conversation is following LGSVL documentation, we might change it later on if necessary.

File: 03-raycast.py (hunk 3 above)
Comment (KunSu) [reply 03-raycast.py:48]: Most of the time, I do not think it is necessary. But we can add this function to the backlog.

**Review by SungYinYang — CHANGES_REQUESTED** [review]
I think it is better to put what we wrote in a separate folder name "script".
It is a little bit hard to follow at this moment.
Other than that, everything looks good to me.

**Comment by KunSu** [comment]
> I think it is better to put what we wrote in a separate folder name "script".
> It is a little bit hard to follow at this moment.
> Other than that, everything looks good to me.

I tried to put 01-scenario inside a folder named scenarios. However, the script failed to load the config library under the simulation folder due to the path issue. I googled and tried several solutions but none of them are working. I do not want to spend too much time solving this issue. Therefore, I created issue #2 so you can keep track of it or solve it.

**Review by SungYinYang — APPROVED** [review]
Then we will fix it later
